connectivity/check/action.go

```diff
@@ -163,23 +167,39 @@ func (a *Action) ExecInPod(ctx context.Context, cmd []string) {
 	cmdName := cmd[0]
 	cmdStr := strings.Join(cmd, " ")
 
-	if stderr.Len() > 0 {
-		a.test.Debugf("%s stderr: %s", cmdName, stderr.String())
-	} else if stdout.Len() > 0 {
-		a.test.Debugf("%s stdout: %s", cmdName, stdout.String())
-	}
-
 	if err != nil || stderr.Len() > 0 {
 		if a.shouldSucceed() {
 			a.Failf("command %q failed: %s", cmdStr, err)
 		} else {
-			a.test.Debugf("command %q failed as expected: %s", cmdStr, err)
+			a.test.Infof("command %q failed as expected: %s", cmdStr, err)
```

Comment:
I'm not sure if expected conditions need to be logged as Info, imo debug was appropriate here. `cmdStr` is usually rather long and really clutters the output when running verbose.

Reply:
Changed back to debug, but added stderr to the debug output also when failing as expected.